src/content/docs/release-notes/docs-release-notes/docs-29-05-2024.mdx

```diff
@@ -0,0 +1,26 @@
+---
+subject: "Docs"
+releaseDate: '2024-12-06'
+version: 'version: November 29-December 5, 2024'
+---
+
+### New docs
+
+* [Gain security insights](https://docs.newrelic.com/whats-new/2024/12/whats-new-12-03-fossa/) with FOSSA in New Relic Vulnerability Management dashboard. Quickly view the FOSSA open-source vulnerability scan results alongside other security detections within the New Relic Vulnerability Management dashboard.
```

Comment:
FOSSA integration is already part of Docs version: November 1-7, 2024. This week we just added the What's new section for it. Please update it accordingly.

My understanding is, we should not cover it in the release note.